database/migrations/2023_09_12_162426_create_products_table.php

```diff
@@ -15,7 +15,7 @@ public function up(): void
             $table->foreignUuid('brand_id')->constrained();
             $table->foreignUuid('color_id')->nullable()->constrained();
             $table->foreignUuid('size_id')->nullable()->constrained();
-            $table->boolean('status')->default(1)->comment('1:active, 0:archived');
+            $table->enum('status', ['active', 'archived', 'received'])->default('active');
```

Comment:
don't change in existing migration file. Now onwards give separate migration for column add/edit/remove/rename etc..

Reply:
okk

Reply:
then revert back the changes